src/web/InputHistory.ts

```diff
@@ -16,7 +18,7 @@ export default class InputHistory {
 
   debounceTime: number;
 
-  constructor(depth: number, debounceTime = 150) {
+  constructor(depth: number, debounceTime = TEST_CONST.INPUT_HISTORY_DEBOUNCE_TIME_MS) {
```

Comment:
Source code files shouldn't rely on test configs.

Reply:
initially, those were constants for general use, and the value for this constant was taken from this place. Michał suggested renaming those constants as test constants, so I can either make them standard constants as intended or revert this change and affect test reliability - someone might change it in the future and we might get different results because tests won't have debounce updated.